code/modules/shuttle/on_move.dm

```diff
@@ -330,10 +330,11 @@ All ShuttleMove procs go here
 /atom/movable/light/onShuttleMove(turf/newT, turf/oldT, rotation, list/movement_force, move_dir, old_dock)
 	return FALSE
 
-/obj/docking_port/stationary/onShuttleMove(turf/newT, turf/oldT, rotation, list/movement_force, move_dir, old_dock)
+/obj/docking_port/stationary/onShuttleMove(turf/newT, turf/oldT, rotation, list/movement_force, move_dir, obj/docking_port/stationary/old_dock)
 	if(old_dock == src) //Don't move the dock we're leaving
 		return FALSE
-
+	if(old_dock && !old_dock.can_move_docking_ports_from)
+		return FALSE
```

Comment:
Instead of this make a var for stationary docks that disallows movement entirely

Reply:
Stationary docking ports could not move in the past, but this was changed at some point in the shuttle code rewrites. I did it this way to preserve the current behavior.

Reply:
The behaviour was changed intentionally, I'm the main one handling docking changes and issues atm. A var for preventing individual movement is better here than stationary docks preventing all movement of other stationary dock.

Reply:
The only docking port with can_move_docking_ports_from set to false is the custom one placed by the shuttle docking computer. All other types of docking ports will behave exactly the same way that they do now.